rmi/servercode/ResImpl/MiddleWareImpl.java

```diff
@@ -91,6 +93,22 @@ public static void main(String[] args) {
 
         // Setup transaction manager
         TM = new TransactionManager();
+
```

Comment:
When we spin up the MW, we restore all of our RM hashtables to those found on the master record